ivadomed/utils.py

```diff
@@ -387,8 +399,10 @@ def segment_volume(folder_model, fname_image, fname_roi=None, gpu_number=0):
     if fname_roi is None and 'filter_empty_mask' in loader_params["slice_filter_params"]:
         print("\nWARNING: fname_roi has not been specified, then the entire volume is processed.")
         loader_params["slice_filter_params"]["filter_empty_mask"] = False
+        filename_pairs = [([fname_image], None, None, [{}])]
+    else:
+        filename_pairs = [([fname_image], None, [fname_roi], [{}])]
 
-    filename_pairs = [([fname_image], [None], fname_roi, [{}])]
```

Comment:
This line should work what was the issue?

Reply:
Might be fixed in new PR but it used to fails because [None] was not considered as None.